### PR TITLE
Improve Github Workflows

### PR DESCRIPTION
**Pull request by FaserF**
# Proposed Changes
This PR should fix an issue, where Megalinter scanned workflow files, despite being excluded.
Also this PR should fix the issues, that Megalinter had found within the Yaml files.

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

**Comment by FaserF** [comment]
GitHub Actions contain workflow_dispatch inputs parameters
To maintain an untampered workflow, pipelines should be unaffected by user input. In GitHub Actions, workflow_dispatch allows you to manually trigger pipelines and enter unique inputs per run. While this may be helpful for running different scenarios, it breaks the policy that workflows should not use user input and should be automated.

Thats the reason. Github Actions wont like manual user input in those runs.

We will remove the check fully for now

**Comment by FaserF** [comment]
About the part that megalinter still scans files within the workflow folder: Looks like a bug to me, but I will try to figure it out

**Comment by Romanitho** [comment]
> GitHub Actions contain workflow_dispatch inputs parameters To maintain an untampered workflow, pipelines should be unaffected by user input. In GitHub Actions, workflow_dispatch allows you to manually trigger pipelines and enter unique inputs per run. While this may be helpful for running different scenarios, it breaks the policy that workflows should not use user input and should be automated.
> 
> Thats the reason. Github Actions wont like manual user input in those runs.
> 
> We will remove the check fully for now

I agree with that, but I implemented a workload that I want to run with parameters to automatically create a new release (and avoid any mistake on releasing new version)

File: .github/workflows/WAU-CreateNewVersion.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-        - Patch
-        - Minor
-        - Major
+          - Patch
+          - Minor
+          - Major
+        required: true
       pre-release:
         type: boolean
         description: Set as Pre-release version
```
Comment (Romanitho) [inline .github/workflows/WAU-CreateNewVersion.yml:18]: Thanks for your suggestion ;)

**Pull request merged**